feature-libs/product-configurator/rulebased/styles/_configurator-attribute-read-only.scss

```diff
@@ -1,5 +1,6 @@
 %cx-configurator-attribute-read-only {
   @include cx-configurator-attribute-type();
+  @include cx-configurator-form-group();
```

Comment:
Breaking change, needs to be equipped with feature flag. See https://sap.github.io/spartacus-docs/css-architecture/#style-versioning